CS8785 due to missing `System.Text.Encodings.Web` assembly in Dev16.11

## Actual behavior

Building the project fails to generate the sources when a NativeMethods.json file exists.

The following warning is emitted:
```
CSC : warning CS8785: Generator 'SourceGenerator' failed to generate source. It will not contribute to the output and compilation errors may occur as a result. Exception was of type 'FileNotFoundException' with message 'Could not load file or assembly 'System.Text.Encodings.Web, Version=4.0.5.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The system cannot find the file specified.'
```

## Expected behavior

The sources are successfully generated during the build.

## Repro steps

1. `NativeMethods.txt` content:
```
CopyFileEx
CoTaskMemFree
ExtractIconEx
FileOpenDialog
FileSaveDialog
GetActiveWindow
GetModuleFileName
GetSystemWindowsDirectory
GetWindowLongPtr
HRESULT_FROM_WIN32
IFileDialog
IShellItem2
ITaskbarList3
SendMessage
SetWindowLongPtr
SHCreateItemFromParsingName
ShellExecuteEx
SHGetFileInfo
SHGetKnownFolderItem
SHGetStockIconInfo
SHOpenFolderAndSelectItems
SHParseDisplayName
StrCmpLogical
TaskbarList
TaskDialogIndirect

E_*
FOLDERID_*
ICON_*
MAX_PATH
MESSAGEBOX_RESULT
PBST_*
S_*
SEE_MASK_*
SHOW_WINDOW_CMD
TASKDIALOG_ELEMENTS
TASKDIALOG_MESSAGES
TD_*
WIN32_ERROR
WINDOW_EX_STYLE
WM_*
```

2. `NativeMethods.json` content (if present):
```json
{
  "$schema": "https://aka.ms/CsWin32.schema.json",
  "comInterop": {
    "preserveSigMethods": [
      "IFileDialogEvents"
    ]
  }
}
```

3. Any of your own code that should be shared?

### Context

- CsWin32 version: 0.3.49-beta
- Win32Metadata version (if explicitly set by project): 59.0.13-preview
- Target Framework: net5.0-windows
- `LangVersion` (if explicitly set by project): -

--- a/Directory.Packages.props
+++ b/Directory.Packages.props
@@ -31,7 +31,7 @@
     <PackageVersion Include="System.Memory" Version="4.5.5" />
     <PackageVersion Include="System.Reflection.Metadata" Version="7.0.0" />
     <PackageVersion Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0" />
-    <PackageVersion Include="System.Text.Encodings.Web" Version="4.7.2" />
+    <PackageVersion Include="System.Text.Encodings.Web" Version="4.7.1" />
     <PackageVersion Include="System.Text.Json" Version="4.7.2" />
     <PackageVersion Include="Xunit.Combinatorial" Version="1.6.24" />
     <PackageVersion Include="xunit.runner.visualstudio" Version="2.5.7" />

--- a/integration-tests/nonsdk/NativeMethods.json
+++ b/integration-tests/nonsdk/NativeMethods.json
@@ -1,0 +1,3 @@
+{
+  // This file may be blank, but it exercises the analyzer's System.Text.Json and dependencies.
+}

--- a/integration-tests/sdk/NativeMethods.json
+++ b/integration-tests/sdk/NativeMethods.json
@@ -1,0 +1,3 @@
+{
+  // This file may be blank, but it exercises the analyzer's System.Text.Json and dependencies.
+}

